ftest/nightwatchjs/tests/smoke_test.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+    'Sample Smoke Test' : function (browser) {
+        var nuxeo = browser.page.nuxeo();
+        nuxeo.navigate()
+        .waitForElementVisible('body', 1000)
+        .setValue('@username', 'Administrator')
+        .setValue('@password', 'Administrator')
+        .click('@submit')
+        .waitForElementVisible('@welcomemsg', 5000)
+        .assert.title('Nuxeo Platform - Domain')
+        .assert.containsText('@welcomemsg', 'Copyright © 2001-2016 Nuxeo and respective authors.');
```

Comment:
Yes totally. Maybe i could check for another thing (maybe more stable than a copyright) or just login, but i wanted to put one example of string checking and I guess in March we will have to check the new webui afterward. So I can comment it for instance if there is a concern about it.